use-timescale/integrations/index.md

```diff
@@ -51,9 +51,11 @@ Some of the most in-demand integrations for $CLOUD_LONG are listed below, with l
 |:--------------------------------:|-------------------------------------------------------------------------------------|
 | [Amazon SageMaker][amazon-sagemaker]| Build, train, and deploy ML models into a production-ready hosted environment.      |
 | [Apache Airflow][apache-airflow] | Programmatically author, schedule, and monitor workflows.                           |
+| [Apache Kafka][kafka] | A distributed event streaming platform used for high-performance data pipelines, streaming analytics, and data integration. |
```

Comment:
This needs to be in a verbal form, like we agreed for all of them

Reply:
Done